notebooks/official/feature_store/vertex_ai_feature_store_iam_policy.ipynb

```diff
@@ -0,0 +1,708 @@
+{
```

Comment:
<p>Update the links</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2913/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Fixed. The notebook should appear in main once the PR is merged.</p>